package.json

```diff
@@ -20,14 +20,16 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.7.15",
+    "@inrupt/jest-jsdom-polyfills": "^2.1.1",
     "copy-to-clipboard": "^3.3.3",
     "minidenticons": "^3.1.2",
     "react": "^18.2.0",
-    "react-currency-input-field": "^3.6.10",
+    "react-currency-input-field": "^3.6.11",
     "react-dom": "^18.2.0",
     "react-hot-toast": "^2.4.1",
     "react-icons": "^4.8.0",
     "tw-colors": "^1.2.5",
+    "viem": "^1.5.2",
```

Comment:
I don't think we need for now. Are you not using at least...

Reply:
yes i'm using it here with the format units fn: 
```

 const formattedBalance = formatValue({
    value: formatUnits(bigintBalance, decimals).toString(),
    groupSeparator: ',',
    decimalSeparator: '.',
    decimalScale: 2,
  });
```

Reply:
Ohh ok! Sorry!